lib/std/target.zig

```diff
@@ -237,7 +237,7 @@ pub const Target = struct {
                     .macos => return .{
                         .semver = .{
                             .min = .{ .major = 10, .minor = 13 },
-                            .max = .{ .major = 10, .minor = 15, .patch = 7 },
+                            .max = .{ .major = 11, .minor = 1 },
```

Comment:
FYI there's a [11.2 β](https://developer.apple.com/documentation/macos-release-notes/macos-big-sur-11_2-beta-release-notes).

Reply:
So far we've just been updating the `.max` as releases come out and a dev has access to a system to very the version. I think that approach is fine for now.